crates/rspack_plugin_devtool/src/lib.rs

```diff
@@ -66,88 +200,231 @@ impl Plugin for SourceMapDevToolPlugin {
     "rspack.SourceMapDevToolPlugin"
   }
 
+  async fn compilation(
+    &self,
+    _args: CompilationArgs<'_>,
+    _params: &CompilationParams,
+  ) -> PluginCompilationHookOutput {
+    // TODO: Temporarily use `devtool` to pass source map configuration information
+    let mut devtool = _args.compilation.options.devtool.lock().await;
+    devtool.add_source_map();
+    if self.filename.is_none() {
+      devtool.add_inline();
+    }
+    if self.source_mapping_url_comment.is_none() {
+      devtool.add_hidden();
+    }
+    if !self.columns {
+      devtool.add_cheap();
+    }
+    if self.no_sources {
+      devtool.add_no_sources();
+    }
+    if self.module {
+      devtool.add_module();
```

Comment:
module should setup by SourceMapDevToolModuleOptionsPlugin

Reply:
I believe it is not the right time to implement `SourceMapDevToolModuleOptionsPlugin`. We should wait for the design to finalize regarding `useSourceMap` and `useSimpleSourceMap` within the module.